src/System.Net.Primitives/tests/PalTests/System.Net.Primitives.Linux.Pal.Tests.csproj

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@pgavlin You are correct and after giving it more thought, I think it's ok to have the PAL tests arranged as they are. This way we create a new integration tests category where the requirements are similar to the UnitTests with the addition that they _can_ make system calls.

I took a brief look through the UnitTests including OS-specific PAL files and if my understanding is correct, the UT definition has _not_ been broken. The tests remain OS agnostic and they do not make any system calls (although they do test OS specific implementations which is ok). 
I assume that the existence of multiple OS-specific csproj files is there only because there would have been type duplicates within the PAL.

What still is unclear to me is why do these run only on certain OSs. Couldn't they be enabled for all OSs?

:+1: LGTM